src/Tokenizer/Tokens.php

```diff
@@ -869,7 +869,27 @@ public function findSequence(array $sequence, $start = 0, $end = null, $caseSens
     public function insertAt($index, $items)
     {
         $items = \is_array($items) || $items instanceof self ? $items : [$items];
-        $itemsCnt = \count($items);
+
+        $this->insertSlices([$index => $items]);
+    }
+
+    /**
+     * @TODO: docs
+     *
+     * @internal
+     *
+     * 5 => tokenA
+     * 7 => [tokenB, tokenC]
+     *
+     * @param mixed $slices
+     */
+    public function insertSlices($slices)
+    {
+        $itemsCnt = 0;
```

Comment:
`$itemsCount` (or `$itemCount`) - the point is not to abbreviate.

Reply:
done